pkgs/games/mindustry/default.nix

```diff
@@ -202,7 +202,6 @@ stdenv.mkDerivation rec {
       --set ALSA_PLUGIN_DIR ${alsa-plugins}/lib/alsa-lib/'' + optionalString enableWayland '' \
       --set SDL_VIDEODRIVER wayland \
       --set SDL_VIDEO_WAYLAND_WMCLASS Mindustry
-    '' + ''
```

Comment:
Oh, oops. No, this isn't quite right either; it makes all the subsequent code conditional on `enableWayland` and not on `enableClient` as intended. We should instead insert an open paren after `enableClient` on line 196 and close paren after the close quote on line 218, I think.